packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -366,6 +373,31 @@ export default class Agent extends EventEmitter<{|
     }
   };
 
+  launchEditor = ({
+    launchEditorEndpoint,
+    fileName,
+    lineNumber,
+  }: LaunchEditorParams) => {
+    fetch(
+      `/${launchEditorEndpoint}?fileName=${fileName}&lineNumber=${lineNumber}`,
```

Comment:
This looks pretty Unix/Mac specific. Am I reading it wrong?

Reply:
It's just a url pointing to the server, so I think it should be fine